### PR TITLE
MGMT-9176:  SNO installations will use OVN by default

### PR DESCRIPTION
**Pull request by jkilzi**
https://issues.redhat.com/browse/MGMT-9176

Reverts openshift-assisted/assisted-ui-lib#1164
The PR was merged into the wrong release

### CODE REVIEW

File: src/common/components/clusterConfiguration/NetworkConfiguration.tsx [hunk 1]
```diff
@@ -131,10 +137,7 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
 
       {children}
 
-      {!isUserManagedNetworking &&
-        !!clusterFeatureSupportLevels &&
-        clusterFeatureSupportLevels['CLUSTER_MANAGED_NETWORKING_WITH_VMS'] === 'unsupported' &&
-        vmsAlert}
+      {!isUserManagedNetworking && isClusterManagedNetworkingWithVMsUnsupported && { vmsAlert }}
```
Comment (batzionb) [inline src/common/components/clusterConfiguration/NetworkConfiguration.tsx:140]: please remove the {} around vmsAlert it's what caused the networking page to crash

Comment (jkilzi) [inline src/common/components/clusterConfiguration/NetworkConfiguration.tsx:140]:
```suggestion
      {!isUserManagedNetworking && isClusterManagedNetworkingWithVMsUnsupported && vmsAlert }
```

File: src/common/components/clusterConfiguration/networkConfigurationValidation.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const getNetworkInitialValues = (
     hostSubnet: getSubnetFromMachineNetworkCidr(selectMachineNetworkCIDR(cluster)),
     vipDhcpAllocation: cluster.vipDhcpAllocation,
     managedNetworkingType: cluster.userManagedNetworking ? 'userManaged' : 'clusterManaged',
-    networkType: cluster.networkType || 'OpenShiftSDN',
+    networkType: cluster.networkType || 'OVNKubernetes',
```
Comment (batzionb) [inline src/common/components/clusterConfiguration/networkConfigurationValidation.ts:37]: is the default supposed to be OVN for all clusters? or just SNO ones

File: src/common/components/clusterConfiguration/NetworkConfiguration.tsx [hunk 3]
```diff
@@ -131,10 +137,7 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
 
       {children}
 
-      {!isUserManagedNetworking &&
-        !!clusterFeatureSupportLevels &&
-        clusterFeatureSupportLevels['CLUSTER_MANAGED_NETWORKING_WITH_VMS'] === 'unsupported' &&
-        vmsAlert}
+      {!isUserManagedNetworking && isClusterManagedNetworkingWithVMsUnsupported && vmsAlert }
```
Comment (jkilzi) [inline src/common/components/clusterConfiguration/NetworkConfiguration.tsx:140]:
```suggestion
      {!isUserManagedNetworking && isClusterManagedNetworkingWithVMsUnsupported && vmsAlert}
```

File: src/common/components/clusterConfiguration/networkConfigurationValidation.ts (hunk 2 above)
Comment (jkilzi) [reply src/common/components/clusterConfiguration/networkConfigurationValidation.ts:37]: yes, that's the whole point of this task

**Review by batzionb — APPROVED** [review]

**Pull request merged**